docs/PREACT_BADGES.md

```diff
@@ -0,0 +1,328 @@
+## Badges
+
+Badges are self-configured in the Searchspring Management Console.
+
+To displays badges the Result card must include the [OverlayBadge](https://searchspring.github.io/snap/#/components-preact?params=%3Fpath%3D%2Fstory%2Fmolecules-overlaybadge--default) and [CalloutBadge](https://searchspring.github.io/snap/#/components-preact?params=%3Fpath%3D%2Fstory%2Fmolecules-calloutbadge--default) components. 
+
+
+### OverlayBadge
+
+The `OverlayBadge` component wraps elements (children) that should have badges overlayed - typically the product image.
+
+```jsx
+<OverlayBadge controller={controller} result={controller.store.results[0]}>
+	<img src='/images/example.png'/>
+</OverlayBadge>
+```
+
+### CalloutBadge
+
+The `CalloutBadge` component displays badges inline and can be placed in any position in the Result card. 
+
+```jsx
+<CalloutBadge result={controller.store.results[0]} />
+```
+
+## Custom Badge Templates
+
+Custom Badge Templates can be created and sync to the Searchspring Management Console using the Snapfu CLI. See [Getting Started > Setup](https://searchspring.github.io/snap/#/start-setup) for installing Snapfu.
+
+### Initialize Custom Badges
+
+First we'll initialize a new custom badge. The code examples on this page will use a `[badgename]` of `CustomBadge`
+
+```sh
+snapfu badges init [badgename]
+```
+This will create two files:
+
+- `src/components/Badges/[badgename]/[badgename].jsx` - The jsx file is the badge component itself that will be displayed by `OverlayBadge` and `CalloutBadge`. The badge `tag`, `value`, and template `parameters` will be passed down as props.
+
+```jsx
+export const CustomBadge = observer((props) => {
+	
+	const { tag, value, parameters } = props;
+	const {} = parameters;
+
+	return (
+		<div className={`ss__badge-custombadge ss__badge-custombadge--${tag}`}>{ value }</div>
+	)
+});
+```
+
+- `src/components/Badges/[badgename]/[badgename].json` - The json file describes the badge template and its parameters. See `Badge Template Parameters` section below for possible parameters.
+
+```json
+{
+	"type": "snap/badge/default",
+	"name": "custombadge",
+	"label": "CustomBadge Badge",
+	"description": "custombadge custom template",
+	"component": "CustomBadge",
+	"locations": [
+		"left",
+		"right",
+		"callout"
+	],
+	"value": {
+		"enabled": true
+	},
+	"parameters": []
+}
+```
+
+### Syncing Custom Badges
+
+Next we'll sync our custom badge - registering it to the Searchspring Management Console. 
+
+```sh
+snapfu badges sync [badgename]
+```
+
+### Using Custom Badges
+
+Finally, in order to use our custom badge component, we'll need to provide a `componentMap` prop containing a mapping of our custom components to the `OverlayBadge` and `CalloutBadge` components.
```

Comment:
Maybe mention that `componentMap` is not necessary when using the default badge types (as a notice).